main.c

```diff
@@ -1061,10 +1061,15 @@ int main(int argc, char **argv)
 	fileidx = options->startnum < filecnt ? options->startnum : 0;
 
 	#if START_FROM_FILE_PATCH
-	if (options->startfile != NULL)
-		for (int i = 0; i < filecnt; ++i)
-			if (strcmp(options->startfile, files[i].path) == 0)
+	if (options->startfile != NULL) {
+		char* startfile = realpath(options->startfile, NULL);
```

Comment:
Return value of `realpath` needs to be checked as it can return `NULL` on error.

Reply:
You also need to `free` the allocation once you're done with it.